pkg/wfc3/calwf3/calwf3/wf3member.c

```diff
@@ -43,10 +42,9 @@ WF3Info *wf3		o: exposure specific flags and info
 
 	/* Initialize local strings */
 	rootname[0] = '\0';
-	outroot[0]  = '\0';
 	mtype[0]    = '\0';
 
-	strcpy (outroot, asn->product[prodid].subprod[posid].name);
+	const char * outroot = asn->product[prodid].subprod[posid].name;
```

Comment:
Changing value of `ACS_FNAME` seems straight-forward enough but what is the reason behind this? If I read this correctly, you are not copying value over anymore, but assign a pointer to `.name`; so theoretically, if `.name` changes or `asn` is removed from memory, then `outroot` points to garbage. Did I read it right?

Best to wait till Megan come back and have a look at this.